src/Aspire.Dashboard/wwwroot/css/app.css

```diff
@@ -54,6 +55,7 @@ fluent-toolbar[orientation=horizontal] {
     --error-ui-accent-foreground-color: #512bd4;
     --reconnection-ui-bg: #D6D6D6;
     --error-counter-badge-foreground-color: #ffffff;
+    --kbd-background-color: var(--fill-color);
```

Comment:
the default color for kbd comes from reboot.css in blazor-fluentui and is inaccessible in dark theme